tests/integration/docusaurus/setup/configuring_data_contexts/how_to_configure_credentials.py

```diff
@@ -2,183 +2,135 @@
 # </snippet>
 """
 
+# NOTE: The following code is only for testing and can be ignored by users.
+
 # Override without snippet tag
 config_variables_yaml = """
-my_postgres_db_yaml_creds:
-  drivername: postgresql
-  host: localhost
-  port: 5432
-  username: postgres
-  password: ${MY_DB_PW}
-  database: postgres
-"""
-
-"""
-# <snippet name="tests/integration/docusaurus/setup/configuring_data_contexts/how_to_configure_credentials.py export_env_vars">
-export POSTGRES_DRIVERNAME=postgresql
-export POSTGRES_HOST=localhost
-export POSTGRES_PORT=5432
-export POSTGRES_USERNAME=postgres
-export POSTGRES_PW=
-export POSTGRES_DB=postgres
-export MY_DB_PW=password
-# </snippet>
+my_postgres_db_yaml_creds: postgresql://postgres:${MY_DB_PW}@localhost:5432/postgres
 """
 
 # Override without snippet tag
 export_env_vars = """
-export POSTGRES_DRIVERNAME=postgresql
-export POSTGRES_HOST=localhost
-export POSTGRES_PORT=5432
-export POSTGRES_USERNAME=postgres
-export POSTGRES_PW=
-export POSTGRES_DB=postgres
+export POSTGRES_CONNECTION_STRING=postgresql://postgres:${MY_DB_PW}@localhost:5432/postgres
 export MY_DB_PW=password
 """
 
 config_variables_file_path = """
-# <snippet name="tests/integration/docusaurus/setup/configuring_data_contexts/how_to_configure_credentials.py config_variables_file_path">
 config_variables_file_path: uncommitted/config_variables.yml
-# </snippet>
 """
 
-datasources_yaml = """
-# <snippet name="tests/integration/docusaurus/setup/configuring_data_contexts/how_to_configure_credentials.py datasources_yaml">
-datasources:
-  my_postgres_db:
-    class_name: Datasource
-    module_name: great_expectations.datasource
-    execution_engine:
-      module_name: great_expectations.execution_engine
-      class_name: SqlAlchemyExecutionEngine
-      credentials: ${my_postgres_db_yaml_creds}
-    data_connectors:
-      default_inferred_data_connector_name:
-        class_name: InferredAssetSqlDataConnector
-  my_other_postgres_db:
-    class_name: Datasource
-    module_name: great_expectations.datasource
-    execution_engine:
-      module_name: great_expectations.execution_engine
-      class_name: SqlAlchemyExecutionEngine
-      credentials:
-        drivername: ${POSTGRES_DRIVERNAME}
-        host: ${POSTGRES_HOST}
-        port: ${POSTGRES_PORT}
-        username: ${POSTGRES_USERNAME}
-        password: ${POSTGRES_PW}
-        database: ${POSTGRES_DB}
-    data_connectors:
-      default_inferred_data_connector_name:
-        class_name: InferredAssetSqlDataConnector
+# add test_data to database for testing
+load_data_into_test_database(
+    table_name="postgres_taxi_data",
+    csv_path="./data/yellow_tripdata_sample_2019-01.csv",
+    connection_string="postgresql://postgres:${MY_DB_PW}@localhost:5432/postgres",
+)
+env_vars = []
+
+# set environment variables using export_env_vars
+for line in export_env_vars.split("export"):
+    if line.strip() != "":
```

Comment:
@Shinnnyshinshin I feel that there can be a cleaner way to parse this -- without the loop (perhaps a regular expression).  Thanks.

Reply:
I agree. I just hardcoded it since there are only 2 variables (which are later taken down)